### ISSUE TITLE
Unable to install from 'bme280wx-0.1.0.zip': no common path

### ISSUE THREAD

**Issue by cibolo** [parautenbach/WeeWX-BME280#1]
Thanks for your hard work...

Having trouble installing, get the following error... any ideas?

`pi@raspberrypi:~ $ sudo wee_extension --install bme280wx-0.1.0.zip
Request to install 'bme280wx-0.1.0.zip'
Extracting from zip archive bme280wx-0.1.0.zip
Traceback (most recent call last):
  File "/usr/share/weewx/wee_extension", line 88, in <module>
    main()
  File "/usr/share/weewx/wee_extension", line 80, in main
    ext.install_extension(options.install)
  File "/usr/share/weewx/weecfg/extension.py", line 121, in install_extension
    "single root directory)" % extension_path)
weecfg.extension.InstallError: Unable to install from 'bme280wx-0.1.0.zip': no common path (the extension archive contains more than a single root directory)
`

**Comment by cibolo** [comment]
Well, fixed that problem with:

sudo wee_extension --install=/home/pi/bme280wx

but now get 
pi@raspberrypi:~ $ wee_config --reconfigure
Using configuration file /etc/weewx/weewx.conf
Enter a brief description of the station, such as its location.  For example:
Santa's Workshop, North Pole
description [XXXX]:

Specify altitude, with units 'foot' or 'meter'.  For example:
35, foot
12, meter
altitude [0, meter]:

Specify latitude in decimal degrees, negative for south.
latitude [90.000]:
Specify longitude in decimal degrees, negative for west.
longitude [0.000]:

You can register your station on weewx.com, where it will be included
in a map. You will need a unique URL to identify your station (such as a
website, or WeatherUnderground link).
Include station in the station registry (y/n)? [n]:

Indicate the preferred units for display: 'metric' or 'us'
units [us]:

Installed drivers include:
  0) ?               (user.bme280wx)           No module named 'smbus2'
  1) AcuRite         (weewx.drivers.acurite)
  2) CC3000          (weewx.drivers.cc3000)
  3) FineOffsetUSB   (weewx.drivers.fousb)
  4) Simulator       (weewx.drivers.simulator)
  5) TE923           (weewx.drivers.te923)
  6) Ultimeter       (weewx.drivers.ultimeter)
  7) Vantage         (weewx.drivers.vantage)
  8) WMR100          (weewx.drivers.wmr100)
  9) WMR200          (weewx.drivers.wmr200)
 10) WMR300          (weewx.drivers.wmr300)
 11) WMR9x8          (weewx.drivers.wmr9x8)
 12) WS1             (weewx.drivers.ws1)
 13) WS23xx          (weewx.drivers.ws23xx)
 14) WS28xx          (weewx.drivers.ws28xx)
choose a driver [4]: 0
Traceback (most recent call last):
  File "/usr/share/weewx/wee_config", line 123, in <module>
    main()
  File "/usr/share/weewx/wee_config", line 117, in main
    config_mgr.run(args, options)
  File "/usr/share/weewx/weecfg/config.py", line 114, in run
    stn_info = self.get_stn_info(config_dict, options)
  File "/usr/share/weewx/weecfg/config.py", line 155, in get_stn_info
    stn_info.update(weecfg.prompt_for_driver_settings(driver, config_dict))
  File "/usr/share/weewx/weecfg/__init__.py", line 1632, in prompt_for_driver_settings
    __import__(driver)
  File "/usr/share/weewx/user/bme280wx.py", line 42, in <module>
    import smbus2
ModuleNotFoundError: No module named 'smbus2'

**Comment by cibolo** [comment]
fixed it with
sudo pip3 install smbus2

**Comment by parautenbach** [comment]
Hi @thecyberdotninja, I'm glad you got it working and thanks for the feedback! 

I wrote this little add-on quickly one afternoon for WeeWX version 3.9.2 (which I'm still running) and hasn't updated it since or gave it much attention otherwise. 

I'm not sure why you had to `sudo` (perhaps I had to do that too, but didn't document it) – at least not during installation. When dealing with hardware interfaces though, one often needs to ensure the system users are in the right groups, but here too I can't remember whether that was necessary. When I upgrade my system one day I need to check and document this. 

As for the `smbus2` dependency: This was missing and I added it now to the README and the add-on's comments in the code. 

Which version of WeeWX are you running and is the add-on now working fine?
